CMake/FindCBLAS.cmake

```diff
@@ -0,0 +1,28 @@
+# Try to find BLAS, including cblas.h
+#
+# Once done this will define
+#  CBLAS_FOUND        - system has a BLAS library
+#  CBLAS_INCLUDE_DIRS - the header directory containing cblas.h
+#  CBLAS_LIBRARIES    - the CBLAS library
+
+# Copyright (c) 2020, Mahrud Sayrafi, <mahrud@umn.edu>
+# Redistribution and use is allowed according to the terms of the BSD license.
+
+find_path(CBLAS_INCLUDE_DIRS NAMES cblas.h
+  HINTS CBLAS_ROOT ENV CBLAS_ROOT
+  PATHS ${INCLUDE_INSTALL_DIR} ${CMAKE_INSTALL_PREFIX}/include
+  PATH_SUFFIXES openblas cblas blis
```

Comment:
```suggestion
```

Reply:
I'm afraid this is necessary, for instance on my machine:
```
[mahrud@noether emacs]$ ls /usr/include/*/cblas.h 
/usr/include/cblas/cblas.h  /usr/include/openblas/cblas.h
```

Reply:
Also, I believe CMake checks to see if the file is on the path alone, and if not tries the suffixes in order.